core/src/mindustry/input/InputHandler.java

```diff
@@ -1213,7 +1213,14 @@ void iterateLine(int startX, int startY, int endX, int endY, Cons<PlaceLine> con
             line.x = point.x;
             line.y = point.y;
             if(!overrideLineRotation || diagonal){
-                line.rotation = next != null ? Tile.relativeTo(point.x, point.y, next.x, next.y) : baseRotation;
+                if (next != null){
```

Comment:
incorrect formatting here and below, see the guidelines

Reply:
fixed